handlers.go

```diff
@@ -120,3 +143,20 @@ func deleteUser(w http.ResponseWriter, r *http.Request) {
 		}
 	}
 }
+
+func getCookieHandler(w http.ResponseWriter, r *http.Request, cookie_name string) int {
+	value, err := cookies.Read(r, cookie_name)
+	if err != nil {
+		switch {
+		case errors.Is(err, http.ErrNoCookie):
```

Comment:
@seoyoungcho213 We shouldn't throw an error when not finding cookie. This should only happen when not finding cookie AND trying to access a non-entry point website such as "dashboard," "settings," "etc..."